backend/app/api/api_v1/endpoints/billing.py

```
@@ -0,0 +1,172 @@
+@router.get("/stripe/config", response_model=BillingStripeConfigResponse)
+async def get_stripe_billing_config(
+    _auth: dict = Depends(require_admin_auth),
+) -> BillingStripeConfigResponse:
+    """Return non-secret Stripe Billing configuration status."""
+    return {"stripe": stripe_public_config()}
+
+
+@router.post("/stripe/checkout", response_model=StripeHostedSessionResponse)
+async def create_checkout_session(
+    payload: StripeCheckoutRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)